handbook/engineering/search/index.md

```diff
@@ -12,50 +12,52 @@ Sourcegraph search is:
+### Scalable
 
-### Scale indexed search to 500k repositories
+- **Scale indexed search to 500k repositories**
    - **Problem:** We have customers who need Sourcegraph to scale to 500k repositories. Some parts of Sourcegraph don't work well at that scale.
    - **Outcome:** Sourcegraph can search 500k repositories in less than 300ms. This is on par with [grep.app](https://grep.app).
    - **Plan:** Incrementally add repositories to Sourcegraph.com until searches get slow or start breaking. Fix those things. Then continue adding repositories.
-   - **Owners:** TBD
+   - **Owners:** Keegan/Stefan
    - **Status:** Sourcegraph.com currently has about 100k repositories.
-
-## Short term (1-3 months)
-
-Our current focus is documented in the [tracking issue for the current milestone](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Atracking+label%3Ateam%2Fsearch).
+   - **Estimated completion**: TODO
 
 ### Fast
 
 - **Unblock improving user perceived performance**
   - **Problem:** Our current search infrastructure relies on on-shot requests with aggressive timeouts, but it is hard to tune those timeouts for large codebases. If we make the timeouts too small, then search may not return results for needle-in-the-haystack queries (because the search timed out). If we make the timeouts too large, then search becomes slower across the board and users spend time waiting for more results when we could have returned useful results to them sooner. Detecting and handling these timeouts across multiple service boundaries is also error prone and has led to bugs.
-  - **Outcome:** 
-      - Unblock the ability to add responsive and fast-loading search results. 
+  - **Outcome:**
+      - Unblock the ability to add responsive and fast-loading search results.
       - Return results faster for large result sets, e.g. indexed repositories in very large codebases.
   - **Plan:** Streaming search
   - **Owner:** Keegan
   - **Status:** In progress
+  - **Estimated completion:** TODO
 
 ### Expressive
 
-- **Search Expressions (AND/OR/NOT)** ([term explanation](https://github.com/sourcegraph/sourcegraph/issues/13126), [term RFC](https://docs.google.com/document/d/1SHky6nodPs1w_zRXz24jB2nq5LbMCJl1u7hcLTGHDL8/edit#))
-  - **Problem:** We have customers who want to migrate from OpenGrok and other search tools, and they want to be able to do searches that are available in those tools.
-  - **Outcomes:** OpenGrok users eagerly migrate to Sourcegraph, and can run complex searches on Sourcegraph.
+- **Extend Search Query Language**
+  - **Problem:** It is imperative that users can find and filter the code, files, repositories, and commits they care about in extremely large codebases.
+  - **Outcomes:** Users can express relations on code, files, repositories, and commits in search queries to more effectively filter the data they need. Sourcegraph is the exclusive industrial-strength search solution that provides these capabilities.
```

Comment:
Follow up on https://github.com/sourcegraph/about/pull/1462#discussion_r480672724:

> Sourcegraph is the only industrial-strength search solution that provides these capabilities.

>> This feels a bit vision-like or something we should aspire to be.

I didn't intend for this to seem vision-like: I see it as a concrete outcome, and one that I'm motivated to deliver within the timeframe here. We have a page that [compares Sourcegraph features to alternatives](https://about.sourcegraph.com/workflow#summary-feature-comparison-chart). Concretely, we can think about the outcome of the work/goal I'm setting here as adding more rows to that feature table. I want to deliver outcomes that set us above alternatives and provides greater value prop for customers. Another way to think of why it makes sense to say this is that, I  so I think it makes sense to add to that as an explicit outcome here.